src/Functions.Worker.Extensions.OpenAI/Embedding/InputType.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Microsoft.Azure.Functions.Worker.Extensions.OpenAI.Embedding;
```

Comment:
Let's use "Embeddings" (plural) instead of "Embedding" as the directory and namespace name. That will be consistent with the class names and how the term is normally used in documentation.

Reply:
So I had that initially, but for some reason the if the namespace is called `Embeddings` that messes with the type `Embeddings` that the OpenAI sdk uses and it doesn't properly compile

Reply:
I changed the sample project to be called `Embeddings` though and have that be consistent. I just didn't change the namespace for the worker/ webjobs projects because of the conflict

Reply:
updated extension namespace to Embeddings.
Resolved conflicts with 

```
using OpenAISDK = Azure.AI.OpenAI;
...
JsonConverter<OpenAISDK.Embeddings>
```